src/python/deepgnn/graph_engine/snark/convert.py

```diff
@@ -39,6 +39,7 @@ def __init__(
         skip_edge_sampler: bool = False,
         file_iterator: Optional[TextFileIterator] = None,
         debug: bool = False,
+        is_temporal: bool = False,
```

Comment:
We should auto-detect this instead to keep it simple for users.

Reply:
How can we autodetect it, if we also want to support default values?